### ISSUE TITLE
2023.4.x breaks this component

### ISSUE THREAD

**Issue by DavidFW1960** [exxamalte/home-assistant-custom-components-nsw-rural-fire-service-fire-danger#59]
```
2023-04-05 09:16:39.399 ERROR (MainThread) [custom_components.nsw_rural_fire_service_fire_danger.coordinator] Unexpected error fetching nsw_rural_fire_service_fire_danger data: 'NoneType' object is not callable
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 250, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 206, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/nsw_rural_fire_service_fire_danger/coordinator.py", line 97, in async_update
    await self._rest.async_update()
  File "/usr/src/homeassistant/homeassistant/components/rest/data.py", line 75, in async_update
    self.data = response.text
  File "/usr/local/lib/python3.10/site-packages/httpx/_models.py", line 578, in text
    decoder = TextDecoder(encoding=self.encoding or "utf-8")
  File "/usr/local/lib/python3.10/site-packages/httpx/_models.py", line 600, in encoding
    encoding = self.default_encoding(self._content)
```

**Comment by exxamalte** [comment]
Thanks for reporting this. I can reproduce this issue with version 2023.4.0.dev0, and using the standard feed (XML based). The extended feed (JSON based) appears to work fine.

**Comment by DavidFW1960** [comment]
Switched to extended and it's ok. But it did rename the sensors which was annoying..

**Comment by exxamalte** [comment]
The fix is included in version 0.15.
